zp-relayer/utils/constants.ts

```diff
@@ -11,6 +11,12 @@ const constants = {
   POOL_ID_SIZE: 6,
   ZERO_ADDRESS: '0x0000000000000000000000000000000000000000',
   INIT_ROOT: '11469701942666298368112882412133877458305516134926649826543144744382391691533',
+  RETRY_CONFIG: {
+    retries: 2,
+    factor: 1.4,
+    maxTimeout: 360000,
```

Comment:
6 minutes seems a bit too much.
maybe 60 seconds?

Reply:
Updated to 60 seconds